client/src/stores/AppStore.js

```diff
@@ -217,8 +217,7 @@ class AppStore extends EventEmitter {
                 this.emit('themeToggle');
                 window.localStorage.setItem('theme', action.theme);
                 break;
-            default:
-                console.log(`[Warning] AppStore invalid action type: ${action.type}`);
+            default: //do nothing
```

Comment:
This works for now, but we may want to refactor so that we aren't using multiple stores for AppStore and RightPaneStore